app/controllers/v1/donations_controller.rb

```diff
@@ -1,9 +1,10 @@
 class V1::DonationsController < V1::BaseController
+  before_action :set_person
```

Comment:
I would avoid this pattern (following thoughtbot's advice). Instead you could just create a method and call it in the create action (`person = create_person` or something like that)